samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -292,6 +320,13 @@ def do_cli(
                 "deployment-branch": deployment_branch,
             }
             pipeline_oidc_provider = GitHubOidcProvider(github_oidc_params, common_oidc_params, GITHUB_ACTIONS)
+        elif oidc_provider == GITLAB:
+            gitlab_oidc_params: dict = {
```

Comment:
I wish there was stronger typing here, it's quite easy to mistype or spend time figuring out the correct names. Can't probably use `TypedDict` since it's 3.8+, but should be able to use the constants from `GitLabOidcProvider` and friends directly.